services-geojson/src/main/java/com/mapbox/geojson/shifter/CoordinateShifter.java

```diff
@@ -0,0 +1,9 @@
+package com.mapbox.geojson.shifter;
+
+public interface CoordinateShifter {
+
+  double shiftLat(double lat);
```

Comment:
I'd suggest the API can be `Coordinate shift(Coordinate lon)`,  case the latitude and longitude used to calculate together. https://github.com/mapbox/mapbox-plugin-ios-china/blob/afbaf3fac65e7ebc243e4a6c2e4a8b85fe858cca/ChinaPlugin/China/Algorithm/gcj.c#L234-L235

Reply:
@lloydsheng Thank you for pointing this out. This will be a pain because we are trying to do this in the constructor.